ee/migrations/0012_migrate_tags_v2.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 3.2.5 on 2022-03-02 22:44
+import traceback
+from typing import Any, List, Tuple
+
+from django.db import migrations
+from django.db.models import Q
+
+from posthog.models.tag import tagify
+from posthog.utils import get_console_logger
+
+
+def forwards(apps, schema_editor):
+    Tag = apps.get_model("posthog", "Tag")
+    TaggedItem = apps.get_model("posthog", "TaggedItem")
+    EnterpriseEventDefinition = apps.get_model("ee", "EnterpriseEventDefinition")
+    EnterprisePropertyDefinition = apps.get_model("ee", "EnterprisePropertyDefinition")
+
+    createables: List[Tuple[Any, Any]] = []
+    batch_size = 1000
+
+    try:
+        # Collect insight tags and taggeditems
+        for [tags, team_id, id] in EnterpriseEventDefinition.objects.exclude(
+            Q(deprecated_tags__isnull=True) | Q(deprecated_tags=[]),
+        ).values_list("deprecated_tags", "team_id", "id"):
+            unique_tags = set([tagify(t) for t in tags if isinstance(t, str) and t.strip() != ""])
+            if unique_tags:
+                for tag in unique_tags:
+                    createables.append((Tag(name=tag, team_id=team_id), TaggedItem(event_definition_id=id)))
+
+        # Collect dashboard tags and taggeditems
+        for [tags, team_id, id] in EnterprisePropertyDefinition.objects.exclude(
+            Q(deprecated_tags__isnull=True) | Q(deprecated_tags=[]),
+        ).values_list("deprecated_tags", "team_id", "id"):
+            unique_tags = set([tagify(t) for t in tags if isinstance(t, str) and t.strip() != ""])
+            if unique_tags:
+                for tag in unique_tags:
+                    createables.append((Tag(name=tag, team_id=team_id), TaggedItem(property_definition_id=id)))
+
+        # Attempts to create tags in bulk while ignoring conflicts. bulk_create does not return any data
+        # about which tags were ignored and created, so we must take care of this manually.
+        tags_to_create = [pair[0] for pair in createables]
```

Comment:
Rather than using just `pair` can we unpack into meaningfully named vars, and use `_` if we do not need to reference, e.g.

```
tags_to_create = [tag for (tag, _) in creatables]
```